test/studies/labelprop/test-extra.json

```diff
@@ -16,4 +16,4 @@
+{ "coordinates": null, "created_at": "Fri Oct 16 16:00:00 +0000 2015", "favorited": false, "truncated": false, "id_str": "12", "entities": { "urls": [ { "expanded_url": null, "url": "https://chapel-lang.org", "indices": [ 69, 100 ] } ], "hashtags": [ ], "user_mentions": [ { "name": "Cray Inc.", "id_str": "11", "id": 11, "indices": [ 25, 30 ], "screen_name": "cray" } ] }, "in_reply_to_user_id_str": null, "text": "Let's mention   the user @cray -- here is an embedded url .......... https://chapel-lang.org", "contributors": null, "id": 12, "retweet_count": null, "in_reply_to_status_id_str": null, "geo": null, "retweeted": false, "in_reply_to_user_id": null, "user": { "profile_sidebar_border_color": "C0DEED", "name": "Cray Inc.", "profile_sidebar_fill_color": "DDEEF6", "profile_background_tile": false, "profile_image_url": "http://some-host.com/some-path.png", "location": "Seattle, WA", "created_at": "Fri Oct 10 23:10:00 +0000 2008", "id_str": "23502385", "follow_request_sent": false, "profile_link_color": "0084B4", "favourites_count": 1, "url": "http://cray.com", "contributors_enabled": false, "utc_offset": -25200, "id": 23548250, "profile_use_background_image": true, "listed_count": 23, "protected": false, "lang": "en", "profile_text_color": "333333", "followers_count": 1000, "time_zone": "Mountain Time (US & Canada)", "verified": false, "geo_enabled": true, "profile_background_color": "C0DEED", "notifications": false, "description": "Cray Inc", "friends_count": 71, "profile_background_image_url": "http://some-host.com/some-path.png", "statuses_count": 302, "screen_name": "gnip", "following": false, "show_all_inline_media": false }, "in_reply_to_screen_name": null, "source": "web", "place": null, "in_reply_to_status_id": null }
```

Comment:
This is the only semi-questionable replacement to me. I think the change should be fine, but I don't understand what this is doing or if it's even used for anything

Reply:
To be honest, I wasn't sure either, but it passed paratest (or at least, didn't fail).  Let's ask @mppf if he would take a look.

Reply:
It's just test data in the Twitter json format. Updating the URL there is fine (as long as the tests pass).

Reply:
Seems to pass.